api/config/config_local.json

```diff
@@ -48,14 +48,14 @@
     "operationalTerraWorkspaceBucket": "fc-56d2f6f5-3efa-46f7-8c01-0911fd77f888",
     "extractionDestinationDataset": "fc-aou-cdr-synth-test-2.wgs_extracted_cohorts",
     "enableJiraTicketingOnFailure": false,
+    "minExtractionScatterTasks": 20,
+    "extractionScatterTasksPerSample": 4,
     "legacyVersions": {
       "gatkJarUri": "gs:\/\/all-of-us-workbench-test-genomics\/wgs\/gatk-package-4.2.0.0-533-g9c2aa67-SNAPSHOT-local.jar",
       "methodNamespace": "aouwgscohortextraction-test",
       "methodName": "GvsExtractCohortFromSampleNames",
       "methodRepoVersion": 15,
-      "methodLogicalVersion": 3,
-      "minExtractionScatterTasks": 20,
```

Comment:
these 2 params are now being used by all workflows, so they belong in the common block